### PR TITLE
Adds ability to load snippet configuration

### PR DESCRIPTION
**Pull request by Marcisbee**
# Description

# Checklist
- [x] I have tested this code locally
- [ ] I have written unit tests
- [x] I looked over the submitted code and I am happy with the quality of it
- [ ] I have updated [the product documentation](https://github.com/kasko/docs)

_Please always fill in all the checkmarks. If some of them are not relevant, then please strikethrough them (~ symbol) and write a comment why it's not relevant._

### CODE REVIEW

File: webpack.dev.js [hunk 1]
```diff
@@ -6,12 +6,14 @@ const merge = require('webpack-merge');
...
 const mockApi = require('@kasko/fe-mock-api-lib');
 const KaskoAssetLoaderPlugin = require('@kasko/webpack-asset-loader-plugin');
+const KaskoWebappSnippetPlugin = require('@kasko/webpack-webapp-snippet-plugin');
 const baseConfig = require('./webpack.common');
 
 const host = process.env.APP_HOST || 'eu1.kaskocloud.com';
 const isDevHost = host === 'dev' || host === 'development';
 const apiUrl = (isDevHost ? 'http://localhost:3000' : `https://api.${host}`);
 const feApiUrl = (isDevHost ? 'http://localhost:3000' : `https://feapi.${host}`);
+const kaskoJsUrl = 'https://eu1.kaskojs.com/v2';
```
Comment (tomstumshais) [inline webpack.dev.js:16]: How you think, maybe we should get this from `env` variables too?

Comment (Marcisbee) [reply webpack.dev.js:16]: I was thinking that this is for a bit advanced use case, as we would only change this when debugging/developing kasko.js locally itself.
Or maybe there are other use cases for this to be changed?

Comment (tomstumshais) [reply webpack.dev.js:16]: Hmm, seems you are right! 👍

File: package.json [hunk 2]
```diff
@@ -21,9 +21,10 @@
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/plugin-proposal-decorators": "^7.4.4",
     "@babel/preset-env": "^7.4.2",
-    "@kasko/fe-mock-api-lib": "^1.0.0",
+    "@kasko/fe-mock-api-lib": "^1.1.0",
     "@kasko/fe-toolbelt": "^1.8.3",
     "@kasko/webpack-asset-loader-plugin": "^1.2.0",
+    "@kasko/webpack-webapp-snippet-plugin": "^1.0.2",
```
Comment (MatissJanis) [inline package.json:27]: FYI: this thing is not public 😄

Comment (Marcisbee) [reply package.json:27]: In registry it is public http://registry.npmjs.org/@kasko/webpack-webapp-snippet-plugin 🤔 
Is it not working?

Comment (MatissJanis) [reply package.json:27]: Ah yes, that one is working. Was trying to sneak a public repo on github :)

**Pull request merged**